flint.ui/src/store/modules/point.js

```diff
@@ -138,27 +140,27 @@ export default {
         }
       }
     },
-    received_data: {},
-    Point_config_pool_1: [],
-    Point_config_pool_2: [],
-    Point_config_pool_3: []
+    results: {},
```

Comment:
Renaming is starting to look good! Please also rename:
* `Point_config` -> `config`